cpr/error.cpp

```diff
@@ -1,11 +1,12 @@
 #include "cpr/error.h"
-#include <curl/curlver.h>
-#include <unordered_map>
 #include <cstdint>
 #include <curl/curl.h>
+#include <curl/curlver.h>
+#include <unordered_map>
 
 namespace cpr {
-static const std::unordered_map<std::int32_t, ErrorCode> curl_error_map = { // NOLINT - (needed because of static init)
+static const std::unordered_map<std::int32_t, ErrorCode> curl_error_map = {
+        // NOLINT - (needed because of static init)
```

Comment:
```suggestion
// NOLINTNEXTLINE - (needed because of static init)
static const std::unordered_map<std::int32_t, ErrorCode> curl_error_map = {
```

That way we get around the formatting issue.